### PR TITLE
Create tdp-51.md

### PR DESCRIPTION
**Pull request by jeshli**
Add TreasuryDAO Charter TDP

### CODE REVIEW

File: TDP/tdp-51.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# TreasuryDAO Charter
```
Comment (dimsome) [inline TDP/tdp-51.md:1]: Please use the header template, see other TDPs.

e.g.:

```
---
tdp: 49
title: Funding Request - mStable Ecosystem subDAO (Sep-Dec 2022)
status: Proposed
author: Fungus (@fungalarmy),  Cam Soulsby (@camsoulsby),  mZeroNine (@mzero2nine)
discussions-to: https://forum.mstable.org/t/funding-request-mstable-ecosystem-subdao-sep-dec-2022/927
created: 2022-08-26
---
```

File: TDP/tdp-51.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Quick Summary
```
Comment (dimsome) [inline TDP/tdp-51.md:3]: We usually use `Simple Summary`

File: TDP/tdp-51.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+This document should serve as a basic pre-draft for the upcoming codified TreasuryDAO charter for the next 12 months (from Governance Resolution of this Charter) under which the TreasuryDAO signers will operate and stand by.
+
+Every sector of the treasury has a particular goal in mind, and a Treasury Charter ensures we are accountable for, and execute on this prescribed goal setting in an open and transparent way.
+The document will contain four major sections, describing in detail the purpose, key activities, additional responsibilities, and relevant sub-committees in detail.
+It is imperative we bring forward a document that can stand the test of time in any kind of market condition, as well as serve as a bookmark of reference if anyone wishes to observe the legitimacy and correct execution of the operational functioning of the DAO in the wider mStable ecosystem.
+Comments and feedback is highly encouraged, and a highly fine-tuned charter will make the life of all participants easier, more enjoyable, and showcase the professionalism under which we steer the Treasury forward.
+
+## Abstract
```
Comment (dimsome) [inline TDP/tdp-51.md:11]: I would make the abstract shorter, it's really just an abstract.

File: TDP/tdp-51.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+The following section will outline the purpose of the TreasuryDAO, as well as some guiding principles under which the signers base their decision-making process on when it comes to managing funds, securing runway, as well as ensuring the appropriate custodianship over the mStableDAO treasury side.
+
+It is the mission of the TreasuryDAO to secure runway to ensure that the mStableDAO Operations are sufficiently funded while investing a portion of its assets into key opportunities in the ecosystem if the budget permits.
```
Comment (dimsome) [inline TDP/tdp-51.md:14]: Please add a `motivation` part. Just outlining quickly why this is important. You can just reshift some of the passages around.

File: TDP/tdp-51.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+As one of its primary goals, the TreasuryDAO wants to conservatively and sustainably grow mStable treasury assets in its custody.
```
Comment (dimsome) [inline TDP/tdp-51.md:15]: This part belongs in `## Specification`

File: TDP/tdp-51.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+It is proposed to have the TreasuryDAO review the Treasury Charter once per fiscal quarter to account for the fast-paced nature of the overall DeFi and crypto ecosystem, during which any amendments and changes will be communicated via our official forum and are subject to a veto process by all participants. Due to the unavoidable lag induced with this, the Treasury will do their best to rely on mainly passive strategies that only require compounding as a primary means of action.
+
+### Key Activities
+This next section outlines the Key Activities the TreasuryDAO performs on a day-to-day basis and highlights the means and ways in which those activities are to be governed and steered when making decisions relating to capital preservation and capital generation alike.
+
+The key activities of the TreasuryDAO can broadly be divided into three categories (General Strategy | Investing | Reporting), which are then further divided into Capital Preservation (securing runway) as well as Capital Generation (Investing into the DeFi ecosystem).
+
+## Treasury Strategy
```
Comment (dimsome) [inline TDP/tdp-51.md:26]: This should be a h3? `###`

File: TDP/tdp-51.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+The following section outlines in detail the strategy that the TreasuryDAO is proposing in order to manage the available funds in the Safe Multisig for both Capital Preservation, and Capital Generation alike.
+
+For the purposes of this section, runway is defined as the number of months that the project can continue to fund operations with a specific pool of funds, based on projected monthly burn-rate. The funds required to achieve a specific length runway will be calculated by multiplying the number of months by the projected burn rate, taking into account subDAO funding proposals, projected protocol expenses and TreausuryDAO and ProtocolDAO operational costs, and conservative revenue estimates.
+
+##### Secure immediate runway (minimum 12 months) (Capital Preservation):
```
Comment (dimsome) [inline TDP/tdp-51.md:31]: Not sure there are #####? Just use bold text if you have to go deeper than h3

**Review by dimsome — CHANGES_REQUESTED** [review]
Please use the consistent structure.

E.g.: https://raw.githubusercontent.com/mstable/MIPs/master/TDP/tdp-36.md

**Comment by jeshli** [comment]
Now consistent  ;P

**Pull request merged**